### PR TITLE
Tiny code cleanup of DownloadInstrument

### PR DESCRIPTION
**Pull request by peterfpeterson**
This is just an occurrence of the [boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)

**To test:**

Look over the changeset.

_There is no associated issue_
_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/DataHandling/src/DownloadInstrument.cpp [hunk 1]
```diff
@@ -87,13 +87,13 @@ void DownloadInstrument::exec() {
   } catch (Mantid::Kernel::Exception::InternetError &ex) {
     std::string errorText(ex.what());
     if (errorText.find("rate limit") != std::string::npos) {
-      g_log.notice() << "Instrument Definition Update: " << errorText << '\n';
+      g_log.notice() << "Instrument Definition Update: " << errorText << "\n";
```
Comment (quantumsteve) [inline Framework/DataHandling/src/DownloadInstrument.cpp]: Why are you changing `'\n'` to "\n"? The former is half the size and slightly more precise.

File: Framework/DataHandling/src/DownloadInstrument.cpp [hunk 2]
```diff
@@ -87,13 +87,13 @@ void DownloadInstrument::exec() {
...
     } else {
       // log the failure at Notice Level
-      g_log.notice() << "Internet Connection Failed - cannot update instrument "
-                        "definitions.\n";
+      g_log.notice("Internet Connection Failed - cannot update instrument "
+                   "definitions.");
```
Comment (quantumsteve) [inline Framework/DataHandling/src/DownloadInstrument.cpp]: 👍

File: Framework/DataHandling/src/DownloadInstrument.cpp [hunk 3]
```diff
@@ -139,7 +139,7 @@ DownloadInstrument::StringToStringMap DownloadInstrument::processRepository() {
                       gitHubJsonDate, Poco::DateTimeFormat::HTTP_FORMAT));
   std::string gitHubInstrumentRepoUrl =
       ConfigService::Instance().getString("UpdateInstrumentDefinitions.URL");
-  if (gitHubInstrumentRepoUrl == "") {
+  if (gitHubInstrumentRepoUrl.empty()) {
```
Comment (quantumsteve) [inline Framework/DataHandling/src/DownloadInstrument.cpp]: 👍

**Comment by quantumsteve** [comment]
LGTM. :shipit:

**Comment by peterfpeterson** [comment]
The failed `SliceViewerPython` test is a red herring and can be ignored.

**Pull request merged**